manifests/init.pp

```diff
@@ -18,16 +18,12 @@
   String                  $bolt_metrics_ensure         = 'present',
   Array[String]           $bolt_hosts                  = puppet_metrics_collector::hosts_with_pe_profile('bolt_server'),
   Integer                 $bolt_port                   = 62658,
-  String                  $activemq_metrics_ensure     = 'absent',
```

Comment:
Should we leave these parameters and warn that they have been deprecated and are no longer used?

Reply:
If we do remove them, it needs to be documented.

Reply:
Not sure, but my inclination was to kill everything amq with fire.

Reply:
I just want to make sure we don't break upgrades because someone has the parameter. Since this is a major version, it is probably fine, but it at least needs to be documented that the parameters are removed.  @MartyEwings What do you think about this? We could move this cleanup to another PR.

Also, we need to clean up those cron jobs if someone upgrades from an old version and doesn't get a version where it was absent.

Reply:
This makes sense, we should 100% document the change,  and I'm on board with separating out, mainly because the changelog will use the PR as the entry and effectively autodocument it "activemq removal"

Reply:
Done in #111